lib/views/status-bar-view.coffee

```diff
@@ -187,6 +204,27 @@ class StatusBarView extends View
                           .text('Success!')
                           .show()
 
+  updateBuildTarget: ->
+    @main.getBuildTargets().then (targets) =>
+      currentBuildTarget = @main.getCurrentBuildTarget()
+      # Clear build target if it doesn't exist for current platform
+      targetExists = targets.reduce (acc, val) =>
+        acc || val.version == currentBuildTarget
+      , false
+      currentBuildTarget = undefined if not targetExists
+
+      if !currentBuildTarget
+        latestVersion = targets.reduce (acc, val) =>
```

Comment:
Totally agree. Once we have the `BuildTargets` collection in `particle-api-js` it could provide  `getLatestVersion(includePrereleases=false)` method. Until then I didn't want to add another three line method that will be deprecated there. IMO we should prioritize cementing `particle-api-js` public interface next sprint and start moving logic there.